server.js

```diff
@@ -6,6 +6,11 @@ var formidable = require('formidable'),
 
 var DIRECT_UPLOAD_LIMIT = 32; // bytes
 
+// convert from UTF-8 to ISO-8859-1
+var LATIN1_SYMBOLS = '¥§©ÆÖÑøøø¼';

```

Comment:
The tests 36 and 37 verify the proper handling of a server Latin1 response.
We could avoid this string constant by changing the tests to pass the symbols in the headers or params but I'm not sure if it will be retrieved correctly (in terms of encoding) by the server and whether it will be needed to be converted to Latin1 after that.

As for the fact that it matches the constant from file-transfer tests - the situation for maintaining this is the same as for cordova_user/cordova_password - or am I missing something?